uploadNeuroDB/NeuroDB/ExitCodes.pm

```diff
@@ -140,7 +140,7 @@ our $TARGET_EXISTS_NO_CLOBBER  = 83; # if tarchive already exists but option
 our $UNKNOWN_PROTOCOL          = 84; # if could not find acquisition protocol
                                      # of the file to be inserted
 our $NOT_A_SINGLE_STUDY        = 85; # if the upload regroups multiple studies
-
+our $BAD_PROTOCOL_SETTING      = 86; # unacceptable mri protocol table entry
```

Comment:
instead of creating a new exit code, I would tend to use PROJECT_CUSTOMIZATION_FAILURE (setting a proper mri_protocol table is part of the customization). Also I guess that eventually the new function you added to check the mri_protocol table will be removed once the second part of the mri_protocol revamp will be merged, right?

Reply:
Okay. I changed the exit code to the existing one.
And yes, the intention is to remove this validation function once the other PR is merged (perhaps a release later) which will then automatically prevent data entry to be non-conform. I added this to the TODO as well. Updated the perldoc as well.

Let me know if any more feedback!